Sources/MessagingPush/MessagingPush.swift

```diff
@@ -31,10 +35,22 @@ open class MessagingPush {
         self.customerIO = customerIO
         self.httpClient = CIOHttpClient(credentials: customerIO.credentials!, config: customerIO.sdkConfig)
         self.jsonAdapter = DITracking.shared.jsonAdapter
+        self.eventBus = DITracking.shared.eventBus
+
+        self.identifyCustomerEventBusCallback = eventBus.register(event: .identifiedCustomer) { event in
```

Comment:
why would we need to `switch` on the event below if it's always of type `.identifiedCustomer`?